lib/contentful.js

```diff
@@ -74,53 +79,57 @@ module.exports.getClient = function getClient() {
  */
 module.exports.contentfulError = function contentfulError(error) {
   const scope = error;
-  scope.message = `Contentful: ${error.message}`;
+  scope.message = `${ERROR_PREFIX} ${error.message}`;
 
   return scope;
 };
 
+module.exports.getFirstItemFromArray = function getFirstItemFromArray(array) {
```

Comment:
`_.extend` does look like a clean way to avoid the disabling `no-param-reassign` lint rules in our middleware.... https://github.com/DoSomething/gambit/blob/c96bdbe67c6b10cdd377f6b182d8b7190f64912b/app/routes/chatbot.js#L241

My vote would be to include and use it instead of our own custom `getFirstItemFromArray` (that could be sense to move to `helpers` if we kept it instead of a different library.. but it sounds like we'd benefit from having the other underscore methods)